docs/source/bottleneck.rst

```diff
@@ -17,15 +17,38 @@ where [args] are any number of arguments to `script.py`, or run
 ``python -m torch.utils.bottleneck -h`` for more usage instructions.
 
 .. warning::
-    Because your script will be profiled, please ensure that it exits in a 
+    Because your script will be profiled, please ensure that it exits in a
     finite amount of time.
 
 .. warning::
     Due to the asynchronous nature of CUDA kernels, when running against
     CUDA code, the cProfile output and CPU-mode autograd profilers may
-    not show correct timings. In this case, the CUDA-mode autograd
-    profiler is better at assigning blame to the relevant operator(s).
+    not show correct timings: the reported CPU time reports the amount of time
+    used to launch the kernels but does not include the time the kernel
+    spent executing on a GPU unless the operation does a synchronize.
+    In this case, the CUDA-mode autograd profiler may be helpful.
```

Comment:
Extra pitfall: ops that synchronize appear to be *extremely* expensive under regular CPU-mode profilers